website/content/docs/commands/operator/generate-root.mdx

```diff
@@ -27,28 +28,155 @@ An unseal key may be provided directly on the command line as an argument to the
 command. If key is specified as "-", the command will read from stdin. If a TTY
 is available, the command will prompt for text.
 
-Please see the [generate root guide](/vault/tutorials/operations/generate-root) for
-step-by-step instructions.
 
-## Examples
+## Generate a root token with one-time password (OTP)
```

Comment:
How-to guides should be standalone docs, not part of a CLI reference page.

That said, I'm struggling to find an appropriate place for the information under the current nav structure. What's the associated task/JTBD for these instructions? Would it make sense to add this guide to the `/vault/docs/auth/token` section in the current docsite?

Reply:
@schavis How about under https://developer.hashicorp.com/vault/docs/concepts/tokens#root-tokens?